modules/ztp-configuring-pgt-image-registry.adoc

```diff
@@ -1,101 +1,175 @@
-<2> Set the appropriate value for `ztp-deploy-wave` as in note 1.
-<3> This assumes that `mount_point` is set to `/var/imageregistry` in `SiteConfig` using StorageClass `image-registry-sc` (see the topic on configuring disk partitioning with `SiteConfig`).
-<4> Set the appropriate value for `ztp-deploy-wave` as in note 1.
-<5> Configure registry to point to the PVC created above.
-<6> Set the appropriate value for `ztp-deploy-wave` as in note 1.
+<1> Set the appropriate value for `ztp-deploy-wave` depending on whether you are configuring image registries at the site, common, or group level. `ztp-deploy-wave: "100"` is suitable for development or testing because it allows you to group the referenced source files together.
+<2> In `ImageRegistryPV.yaml`, ensure that the `spec.local.path` field is set to `/var/imageregistry` to match the value set for the `mount_point` field in the `SiteConfig` CR.
+
++
+[IMPORTANT]
+====
+Do not set `complianceType: mustonlyhave` for the `- fileName: ImageRegistryConfig.yaml` configuration. This can cause the registry pod deployment to fail.
+====
+
+. Commit the `PolicyGenTemplate` change in Git, and then push to the Git repository being monitored by the GitOps ZTP ArgoCD application.
 
 .Verification
 
-. Check that the `Config` CRD of the group `imageregistry.operator.openshift.io` instance is not reporting errors. Run the following command:
+Use the following steps to troubleshoot errors with the local image registry on the managed clusters:
 
-. Check that the `PersistentVolumeClaim` on the managed cluster is populated with data. Run the following command:
+* Verify successful login to the registry while logged in to the managed cluster. Run the following commands:
 
-. Check that the `registry*` pod is up correctly located under the `openshift-image-registry` namespace.
+.. Export the managed cluster name:
++
+[source,terminal]
+----
+$ cluster=<managed_cluster_name>
+----
+
+.. Get the managed cluster `kubeconfig` details:
++
+[source,terminal]
+----
+$ oc get secret -n $cluster $cluster-admin-password -o jsonpath='{.data.password}' | base64 -d > kubeadmin-password-$cluster
+----
 
-. Verify successful login to the registry with `podman`:
+.. Download and export the cluster `kubeconfig`:
 +
 [source,terminal]
 ----
-$ oc login -u kubeadmin -p <password_from_install_log> https://api-int.<cluster_name>.<base_domain>:6443
+$ oc get secret -n $cluster $cluster-admin-kubeconfig -o jsonpath='{.data.kubeconfig}' | base64 -d > kubeconfig-$cluster
 ----
 +
 [source,terminal]
 ----
-$ podman login -u kubeadmin -p $(oc whoami -t) image-registry.openshift-image-registry.svc:5000
+$ export KUBECONFIG=./kubeconfig-$cluster
```

Comment:
[Guidelines](https://redhat-documentation.github.io/modular-docs/#creating-procedure-modules) state one command per step. There are two options here.

1. Break up Download and Export into two individual steps, or
2. Both commands in one code block:
```
[source,terminal]
----
$ podman login -u kubeadmin -p $(oc whoami -t) image-registry.openshift-image-registry.svc:5000 \
	&& export KUBECONFIG=./kubeconfig-$cluster
----

Reply:
Cool, updated.